src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Node<T> {
+    public T item;
+    public Node<T> prev;
+    public Node<T> next;
+
+    public Node(T item, Node<T> prev, Node<T> next) {
+        this.item = item;
+        this.prev = prev;
+        this.next = next;
+    }
+}
```

Comment:
The `Node` class should be an inner class within `MyLinkedList` to improve encapsulation and maintainability, as per the checklist.